autoprotocol/util.py

```diff
@@ -180,10 +180,15 @@ def _check_container_type_with_shape(container_type, shape):
                 f"1 or divisible by 2, but {shape} was specified."
             )
 
-    if shape["format"] == "SBS384" and container_wells < 384:
+    if (
+        shape["format"] == "SBS384"
+        and container_wells < 384
+        and "sbs384_compatible" not in container_type.capabilities
```

Comment:
container type capabilities are usually the names of the operations/instructions that can be performed on a container type. `sbs384_compatible` will never be found in the container_type.capabilities unfortunately.

Reply:
@EribertoLopez In my other feature branch I added an is_reservoir attribute to the container_type dictionary with default of false to be set to true when writing client protocols. This approach above is per Vanessa's suggestion: [https://github.com/autoprotocol/autoprotocol-python/pull/299#discussion_r621251956](url)

In light of these two approaches, do you have an alternative idea?

Reply:
And thanks for the feedback!